Lesson32/Deniz/my-app/src/shared-components/typography/HeadingL/styles.css

```diff
@@ -0,0 +1,9 @@
+h2 {
```

Comment:
It would be better if you also used a class selector in this case to keep your code consistent.